examples/vl-examples.json

```diff
@@ -3,16 +3,28 @@
     {
       "name": "bar_layered_transparent",
       "title": "Bar chart with transparent layers",
-      "showInEditor": true
+      "showInEditor": true,
```

Comment:
Feel free to delete all of the "showInEditor" entries. The examples that have showInGallery:false can be deleted.